explorer/src/pages/TransactionDetailsPage.tsx

```diff
@@ -29,8 +29,18 @@ import { intoTransactionInstruction } from "utils/tx";
 import { TokenDetailsCard } from "components/instruction/token/TokenDetailsCard";
 import { FetchStatus } from "providers/cache";
 
-type Props = { signature: TransactionSignature };
-export function TransactionDetailsPage({ signature: raw }: Props) {
+const AUTO_REFRESH_TIMEOUT = 2000;
+const ZERO_CONFIRMATION_BAILOUT = 20;
```

Comment:
40s of retrying feels like a lot to get to one confirmation. We typically see about 1.5s for the first confirmation. I think 5 retries is sufficient.